### ISSUE TITLE
[COGNITION]: GIBCT - Caution flag headings SHOULD be nested properly

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#7983]
**Feedback framework**

- **⚠️Should**  fix this issue 1-3 sprints post launch
- [508 defect 2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Description

The full alerts on a school profile page are `<h3>` but are nested under an `<h3>` and should be `<h4>` for better searching and understanding for screen readers. Screenshot attached below.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] All full alerts have an `<h4>` heading
- [ ] Screen reader headings menu shows the headings nesting properly
- [ ] Current visual design is maintained

## WCAG or Vendor Guidance (optional)

* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

## Screenshots or Trace Logs

![Screen Shot 2020-04-14 at 8 01 35 AM](https://user-images.githubusercontent.com/934879/79228319-d83db080-7e26-11ea-87f9-61d7470b22aa.png)

**Comment by zurbergram** [comment]
https://github.com/department-of-veterans-affairs/vets-website/pull/12217

**Comment by sonntag-adam** [comment]
![Screen Shot 2020-04-15 at 10.48.24 AM.png](https://images.zenhubusercontent.com/5c98df9c534e09000148f8d1/5d97e5da-92c0-481d-86a5-f8e60e6caa6e)

**Comment by sonntag-adam** [comment]
![Screen Shot 2020-04-15 at 11.05.11 AM.png](https://images.zenhubusercontent.com/5c98df9c534e09000148f8d1/9645b31c-7183-4514-9050-06ed0524c9e2)

**Comment by sonntag-adam** [comment]
@1Copenut in staging.

**Comment by 1Copenut** [comment]
@sonntag-adam Verified in staging. Moving to close.
